javascript/index.js

```diff
@@ -23,18 +23,122 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
-
-
+  }, (error) => console.log(error)); */
 
 // Iteration 1 - using callbacks
-// ...
+mashedPotatoes.splice(4,1,'Mashed potatoes are ready!')
+getInstruction("mashedPotatoes", 0, (step0) => {
```

Comment:
To make the steps appear in right order you need to keep calling the next steps inside the last step:

```
getInstruction("mashedPotatoes", 0, (step0) => {document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`;
    getInstruction( "mashedPotatoes", 1, (step1) => {document.querySelector( "#mashedPotatoes").innerHTML += `<li>${step1}</li>`;

     (...)

        )},
      (error) => console.log(error)
    )},
  (error) => console.log(error)
);
```